client/src/components/relatedProducts/ProductCard.jsx

```diff
@@ -1,27 +1,68 @@
 import React from 'react';
+import EachCard from './EachCard.jsx';
 
-class ProductCard extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      placehoder: []
+const ProductCard = (props) => {
+
+
+  let combine = props.detailInfo.reduce((map, value) => {
+    map[value.product_id] = value;
+    return map;
+  }, {});
+  let newArry = [];
```

Comment:
Is there a better name for this variable instead of newArry? If not, can we change it to newArray?